src/createServer.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const express = require('express');
+const { userRouter } = require('./routes/users.js');
+const { expensesRouter } = require('./routes/expenses.js');
+
+const jsonMiddleware = express.json();
```

Comment:
```suggestion
```

Reply:
already done it above, please see fix above